scripts/Utility.js

```diff
@@ -17,4 +17,24 @@ function formatHour(time) {
 		return time + 'a';
 	}
 	return (time - 12) + 'p';
-}
\ No newline at end of file
+}
+
+function getHolidays() {
```

Comment:
Yup, songs like this one for Christmas - https://www.youtube.com/watch?v=0aCYkAOvmFo

It also feels strange to force the user to listen to the same song over and over again for an entire day without giving them the option to turn it off without disabling KK entirely.